frontend/src/routes/home/Home.tsx

```diff
@@ -52,47 +55,53 @@ function Home(props) {
     return nextDays
   }
 
-  return content.loading ? (
-    <div className="spinner-border" role="status" />
-  ) : (
-    <div className="home-page">
-      <div className="fullScreen-wrapper">
-        <div className="left-title-section">
-          <span className="bold-title">{content.title}</span>
-          <div className="parag">{content.description}</div>
-          <div className="default-red-button">
-            <Link to="/program" onClick={() => setActiveRoute('program')}>
-              AFLĂ PROGRAMUL
-            </Link>
+  const returnContent = () => {
+    return (
+      <>
+        <div className="fullScreen-wrapper">
+          <div className="left-title-section">
+            <span className="bold-title">{content.title}</span>
+            <div className="parag">{content.description}</div>
+            <div className="default-red-button">
```

Comment:
Too generic className, and also specifying the color. You could do something like `program-link` or `program-btn`